src/app/dataset-view/additional-components/flows-component/components/flows-table/flows-table.component.html

```diff
@@ -146,17 +166,89 @@
         </td>
     </ng-container>
 
+    <ng-container matColumnDef="dataset" *ngIf="accountView">
```

Comment:
Test if list of columns allows dataset names instead, you have this data there already.

Reply:
Fixed